setup.py

```diff
@@ -36,6 +36,7 @@
             "teams = lms.djangoapps.teams.plugins:TeamsTab",
             "textbooks = lms.djangoapps.courseware.tabs:TextbookTabs",
             "wiki = lms.djangoapps.course_wiki.tab:WikiTab",
+            "msp_dashboard = msp_assessment.msp_dashboard.views:MspDashboardTab",
```

Comment:
Why we keep this Tab in views ? It would be great if you move it to more specific place. 

Maybe in `tabs.py`